backend/api/v1/endpoints/research.py

```
@@ -9,14 +9,15 @@
 from models.user import User
 from models.research_collaboration import ResearchCollaboration
 from models.collaboration_request import CollaborationRequest
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)